docs/features/map-tiles-custom-areas/high-level-design.md

```diff
@@ -0,0 +1,37 @@
+# Role base access control - High-level design
```

Comment:
also, and very minor - would you mind wrapping the paragraphs to fit within 80cols?

I usually alt+q my way through paragraphs in vscode... there must be a less of caveperson way but since this is good enough for me I haven't bothered checking 😬 . `npx prettier --parser markdown --prose-wrap always -w README.md` will also do - maybe we should add a lint task for all our markdown 🤔